constance/templates/admin/constance/change_list.html

```diff
@@ -48,7 +48,7 @@
 
             {% if fieldsets %}
               {% for fieldset in fieldsets %}
-              <fieldset class="module{% if fieldset.collapse %} collapse{% endif %}">
+              <fieldset class="module{% if fieldset.collapse %} collapse{% endif %} card card-body">
```

Comment:
This is not from standard django.

Reply:
ah ok, yes we have bootstrap in there I guess. Feel free to close this pull request and open an issue if the bad styling actually appears also on other instances.

will come back to it hopefully later but not have time to work on it right now, sry